README.md

```diff
@@ -25,6 +25,32 @@ may eventually evolve into becoming a dynamic server.
 
 To get a local version of the site running, you'll need to have [git](http://git-scm.com/) and [node](http://nodejs.org/) installed on your local machine.
 
+For best results, use node version 0.12.7 (as of November 2015).  Verify your node version using:
+
+```
+node --version
+```
+
+If you need to install node, or switch to a different version of node, Node Version Manager (nvm) is a helpful tool.
+To use nvm for installing and managing different versions of node, first install prerequisite libraries using:
```

Comment:
Good point - and I agree it would be clearer to have some guidance early in the README directing potential contributors to different sections depending on their development platform.  I'm happy to revise and was thinking of having one section for linux (ubuntu, debian and Mac OS), one for windows, and possibly 'other'... having seen someone try to build on a chromebook (!).  Using nvm seems to make the local build process fairly consistent on linux/Mac but I will double check and update.  Let me know if you have additional thoughts on headings or different approaches - thanks!